stackit/internal/services/resourcemanager/project/resource_test.go

```diff
@@ -166,22 +206,72 @@ func TestToCreatePayload(t *testing.T) {
 			&resourcemanager.CreateProjectPayload{
 				ContainerParentId: nil,
 				Labels:            nil,
+				Members:           nil,
+				Name:              nil,
+			},
+			true,
+		},
+		{
+			"mapping_with_conversions_ok",
+			&Model{
+				ContainerParentId: types.StringValue("pid"),
+				Name:              types.StringValue("name"),
+				Members: types.ListValueMust(types.ObjectType{AttrTypes: memberTypes}, []attr.Value{
+					types.ObjectValueMust(
+						memberTypes,
+						map[string]attr.Value{
+							"subject": types.StringValue("owner_email"),
+							"role":    types.StringValue("owner"),
+						},
+					),
+					types.ObjectValueMust(
+						memberTypes,
+						map[string]attr.Value{
+							"subject": types.StringValue("reader_email"),
+							"role":    types.StringValue("reader"),
+						},
+					),
+				}),
+			},
+			&map[string]string{
+				"label1": "1",
+				"label2": "2",
+			},
+			&resourcemanager.CreateProjectPayload{
+				ContainerParentId: utils.Ptr("pid"),
+				Labels: &map[string]string{
+					"label1": "1",
+					"label2": "2",
+				},
 				Members: &[]resourcemanager.Member{
 					{
-						Role:    utils.Ptr(projectOwner),
-						Subject: utils.Ptr("service_account_email"),
+						Subject: utils.Ptr("owner_email"),
+						Role:    utils.Ptr("owner"),
+					},
+					{
+						Subject: utils.Ptr("reader_email"),
+						Role:    utils.Ptr("reader"),
 					},
 				},
-				Name: nil,
+				Name: utils.Ptr("name"),
 			},
 			true,
 		},
 		{
```

Comment:
I think its better to keep the test case where only the `OwnerEmail` is provided, since this will still be possible, right?